sql/core/src/test/scala/org/apache/spark/sql/expressions/ExpressionInfoSuite.scala

```diff
@@ -200,6 +200,7 @@ class ExpressionInfoSuite extends SparkFunSuite with SharedSparkSession {
       val clonedSpark = spark.cloneSession()
       // Coalescing partitions can change result order, so disable it.
       clonedSpark.sessionState.conf.setConf(SQLConf.COALESCE_PARTITIONS_ENABLED, false)
+      clonedSpark.sessionState.conf.setConf(SQLConf.ANSI_ENABLED, false)
```

Comment:
do you know which functions have query examples that rely on ANSI off?

Reply:
Yeah, I just added a "ignore" list when ANSI is on